tools/devcluster.yaml

```diff
@@ -47,6 +47,9 @@ stages:
         log:
           level: debug
         enable_cors: true
+        telemetry: 
```

Comment:
remove this before you merge

Reply:
?

Reply:
that's just a note for myself @philipnrmn. Because this is just to change the config to enable otel. This would be on the users side.